(ForumsTeam) Incorrect url hyperlinked

AjayKumar-N opened the pull request:
The hyperlink [troubleshooting articles] in the "Next steps" section points to generic Microsoft Azure Documentation url, but not to Cloud Service troubleshooting docs. Kindly fix this issue.
Could be redirected to these links instead: 
https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-allocation-failures
https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-troubleshoot-roles-that-fail-start

azurecla commented:
Hi __@kareni__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (kareni). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

kareni commented:
completing all update

azurecla commented:
Hi __@kareni__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (kareni). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

AjayKumar-N committed:
(ForumsTeam) Incorrect url hyperlinked

The hyperlink [troubleshooting articles] in the "Next steps" section points to generic Microsoft Azure Documentation url, but not to Cloud Service troubleshooting docs. Kindly fix this issue.
Could be redirected to these links instead: 
https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-allocation-failures
https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-troubleshoot-roles-that-fail-start
